index.html

```diff
@@ -52,6 +52,12 @@
                 </div>
             </div>
         </header>
+        <section class="section">
+            <div class="spinner">
+                <div class="spinner__triangle"></div>
+                <div class="spinner__inner--round"></div>
```

Comment:
why do you need modifier `--round`? do you have anywhere just `spinner__inner` element?

Reply:
okay, we have fixed excessive modifier